packages/vsce/src/SshErrorHandler.ts

```diff
@@ -74,6 +74,23 @@ export class SshErrorHandler {
         return response;
     }
 
+    /**
+     * Checks if an error is a timeout error
+     * @param error The error to check
+     * @returns True if the error is a timeout error
+     */
+    public isTimeoutError(error: Error | string): boolean {
+        const errorMessage = error instanceof Error ? error.message : error;
+        const timeoutMatches = SshErrors.REQUEST_TIMEOUT.matches;
```

Comment:
Dumb question:
Should we check for RPCErrorCodes instead of trying to match string in the error message?

Reply:
Good call, I've implemented this in 33619c32